authentication/utils.py

```diff
@@ -67,17 +67,22 @@ def load_drf_strategy(request=None):
     )
 
 
+def get_md5_hash(value):
+    """Returns the md5 hash object for the given value"""
+    return hashlib.md5(value.lower().encode("utf-8"))
```

Comment:
why are we not returning hexdigest instead of hash_object?

Reply:
Just returning the object, so maybe later we need to get something different from the object instead of hexdigest